servers/rendering/shader_language.cpp

```diff
@@ -5928,13 +5928,15 @@ Error ShaderLanguage::_parse_block(BlockNode *p_block, const FunctionInfo &p_fun
 			if (tk.type != TK_SEMICOLON) {
 				//all is good
 				_set_error("Expected ';' after discard");
+				return ERR_PARSE_ERROR;
 			}
 
 			p_block->statements.push_back(flow);
 		} else if (tk.type == TK_CF_BREAK) {
 			if (!p_can_break) {
 				//all is good
 				_set_error("Breaking is not allowed here");
```

Comment:
```suggestion
				_set_error("'break' is not allowed outside of a loop or 'switch' statement");
```

Reply:
Judging from the [docs](https://docs.godotengine.org/en/3.2/tutorials/shading/shading_reference/shading_language.html#flow-control), it seems those are the only scenarios where it's used, yup.